windows/d2-installer.nsi

```diff
@@ -351,6 +352,15 @@ SectionGroupEnd
 Function DetectVSAndSDK
     ClearErrors
 
+    ReadRegStr $0 HKLM "SOFTWARE\Microsoft\VisualStudio\SxS\VS7" "15.0"
+    StrCpy $1 "VC2017"
+    IfErrors not_vc2017
+        ${LineRead} "$0\VC\Auxiliary\Build\Microsoft.VCToolsVersion.default.txt" "1" $2
```

Comment:
For reference [NSIS 2.46 Users Manual#LineRead](http://anders_k.users.sourceforge.net/public/NSIS/2.46/Manual.html#lineread).